lockup/src/types.rs

```diff
@@ -45,7 +48,7 @@ pub struct LockupInformation {
 }
 
 /// Contains information about the transfers. Whether transfers are enabled or disabled.
-#[derive(BorshDeserialize, BorshSerialize, Deserialize, Serialize)]
+#[derive(BorshDeserialize, BorshSerialize, Deserialize, Serialize, Debug)]
```

Comment:
It seems `Debug` increases the size of the contract a lot. See +57Kb

Reply:
Apparently `Debug` is not the largest offender here. It's serde with tag, in particular this line:
```rust
#[serde(tag = "type", content = "value")]
```

It adds around 30Kb

Reply:
FYI @mikedotexe @chadoh ^